docs/framework/get-started/system-requirements.md

```diff
@@ -24,7 +24,7 @@ manager: "wpickett"
 # .NET Framework System Requirements
 The tables in this topic provide the hardware, operating system, and software requirements for the .NET Framework 4.5 and its point releases (4.5.1 and 4.5.2), the [!INCLUDE[net_v46](../../../includes/net-v46-md.md)] and its point releases (4.6.1 and 4.6.2), and the .NET Framework 4.7. Development environments that enable you to develop applications for the .NET Framework have a separate set of requirements.  
   
- For download information and links, see [Installation Guide](../../../docs/framework/install/guide-for-developers.md).  
+ For download information and links, see [Install the .NET Framework for developers](../../../docs/framework/install/guide-for-developers.md).  
```

Comment:
These preexisted the PR pointing to *guide-for-developers.md*, which is why I didn't consider changing them (i.e., I trusted the author of the topic). I'll search and take a closer :eyes: look at where the guide for devs is linked.